pkg/bff/members.go

```diff
@@ -0,0 +1,39 @@
+package bff
```

Comment:
I figured since this will likely connect to the members endpoint and because these are mostly authenticated requests, that putting them in `members.go` made sense, where as the endpoints in `gds.go` are the unauthenticated requests to the open GDS api.

Reply:
Makes sense to me.